### DISCUSSION TITLE
favicon.ico not found (404)

### DISCUSSION THREAD

**Discussion by LFrank2021** [opendcim/openDCIM#1388] [Q&A] [answered]
While researching for my other problem I saw an error on the Chrome developer console that the favicon.ico is not found (404).
There is one existing in css/images/favicon.ico but it seems to be expected in vhost directory.
One workaround seems to be a link (ln -s) but is that the best way?
Could this directly be a part of the tar-archive?

Thank you for your time!

Kind regards
Frank

**Comment by wilpig** [comment, marked as answer]
We've never created a favicon for opendcim.  The file you're seeing under css/images dates back to one of the first versions of datatables.  https://github.com/opendcim/openDCIM/commit/c6f5193ea83a9594f5eed45ab7f76aef0dacf032  It's not actually referenced anywhere and I'll try to remember to delete it.  As for chrome asking for it, yeah it does that.  Pretty much all modern browsers look for favicon.ico by default to apply the picture to the URL bar and it will get added to bookmarks and the like.

If you're artistic and want to design one, please submit it.
